test_conformance/extensions/cl_khr_command_buffer/command_buffer_test_copy.cpp

```diff
@@ -197,6 +198,74 @@ struct CopyBufferKHR : public BasicCommandBufferTest
     const cl_char pattern_2 = 0x28;
 };
 
+struct CopySVMBufferKHR : public BasicSVMCommandBufferTest
+{
+    using BasicSVMCommandBufferTest::BasicSVMCommandBufferTest;
+
+    cl_int Run() override
+    {
+        cl_int error = clCommandSVMMemFillKHR(
+            command_buffer, nullptr, svm_in_mem(), data_size(), &pattern_1,
```

Comment:
The ordering of `data_size()`, `pattern_1`, then `sizeof(cl_char)` parameters doesn't look right for the header generated declaration of `CommandSVMMemFillKHR`. Think it should be `pattern1`, `sizeof(cl_char)`, `data_size()`
```c
CommandSVMMemFillKHR(
    cl_command_buffer_khr command_buffer, // command_buffer
    cl_command_queue command_queue, // nullptr
    void* svm_ptr, // svm_in_mem()
    const void* pattern, // data_size() ?
    size_t pattern_size,  // pattern_1 ?
    size_t size, // sizeof(cl_char) ?
    cl_uint num_sync_points_in_wait_list,    // 0
    const cl_sync_point_khr* sync_point_wait_list, //nullptr
    cl_sync_point_khr* sync_point,  // nullptr
    cl_mutable_command_khr* mutable_handle // nullptr
);
```